src/frontend/CMakeLists.txt

```diff
@@ -35,6 +35,12 @@ endfunction(html_compile filename)
 html_compile(${CMAKE_CURRENT_SOURCE_DIR}/frontend.html)
 html_compile(${CMAKE_CURRENT_SOURCE_DIR}/index.html)
 
+# Lint the source JavaScript
+find_program(JS_LINTER jshint)
+if (JS_LINTER)
+  add_custom_target(_lint ALL COMMAND jshint --exclude "'js/lib/*'" . WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Replace the hardcoded ``jshint`` with the one that was found (i.e. ``${JS_LINTER}``)